enterprise/app/controllers/api/v1/accounts/audit_logs_controller.rb

```diff
@@ -4,7 +4,13 @@ class Api::V1::Accounts::AuditLogsController < Api::V1::Accounts::BaseController
   before_action :fetch_audit
 
   def show
-    render json: @audit_logs
+    @audit_logs = @audit_logs.page(params[:page]).per(params[:per_page])
```

Comment:
let's make `params[:per_page]` a constant since that's the pattern which we use everywhere.  
ref: https://github.com/chatwoot/chatwoot/blob/040e9a732f143a73fb8ece6f2583311b03c06a35/app/controllers/api/v1/accounts/contacts_controller.rb#L11

Reply:
Done